hack/update-generated-completions.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+STI_ROOT=$(dirname "${BASH_SOURCE}")/..
```

Comment:
We do:

- https://github.com/openshift/source-to-image/blob/master/hack/test-integration.sh#L7
- https://github.com/openshift/source-to-image/blob/master/hack/build-test-images.sh#L9